CMakeLists.txt

```diff
@@ -101,27 +123,39 @@ macro(jerry_add_link_flags)
   jerry_add_flags(LINKER_FLAGS_COMMON ${ARGV})
 endmacro()
 
+# build mode specific compile/link flags
+if(USING_GCC OR USING_CLANG)
+set(CMAKE_C_FLAGS_RELEASE "-Os")
+endif()
+
```

Comment:
Removed...  I had a little trouble getting the rebase done (and I had a failure in the runtime tests, which was weird, as it went away over the course of the afternoon :-), but it should be ready to go now...